profiles/scraper.py

```diff
@@ -351,7 +379,11 @@ def get_profile_by_url(self, profile_url, company="", role=""):
         If the profile doesn't have experience or education fields we need, then do not store into DB.
         :param profile_url: Person's profile URL that we will be extracting information from.
         :return: True if we have stored the profile into DB, False if we did not store.
-        """ 
+        """
+
+        # visiting check_profile_link, check twice
+        if self.check_profile_link(profile_url) == False and self.check_profile_link(profile_url) == False:
```

Comment:
check_profile_link reloads the page, the second time it is called reloads the page and all progress from the first call is cancelled.